pkg/storageincentives/agent.go

```diff
@@ -298,8 +298,23 @@ func (a *Agent) claim(ctx context.Context) error {
 	}
 
 	if isWinner {
+		a.nodeState.SetState(winner)
```

Comment:
these states do not show much info. for example, if we win one round, do we want to forever show that we won in some round without other info?
maybe you want something like `lastWonRound` instead of permanently having a state of 'winner'.

Reply:
How about we set `a.nodeState.SetState(idle)` somewhere here https://github.com/ethersphere/bee/blob/776a000954c03261c8f6a4a07ef131398bb0524c/pkg/storageincentives/agent.go#L260

Then it won't show winner forever.

Reply:
it's not just about showing it forever, it's about giving users meaningful, useful info.
I propose adding a new `lastWonRound` field